magicbyirineu/Modules/Favorites/Presenter/FavoritesPresenter.swift

```diff
@@ -7,8 +7,9 @@
 //
 
 import Foundation
+import UIKit
 
-class FavoritesPresenter{
+class FavoritesPresenter:NSObject{
```

Comment:
NSObject?

Reply:
Sim, caso contrário a presenter não poderia implementar o protocolo , dando o erro: Type 'FavoritesPresenter' does not conform to protocol 'NSObjectProtocol'